add criteria for expedited schema review process (#69)
HumanCellAtlas/dcp2: docs/dcp2_operating_procedures.rst

expedited review:

- Additions of enum values to ``project.hca_bionetworks.hca_tissue_atlas``

> - Fixing typos, formatting, incorrect urls

Author: ```suggestion
- Fixing typos, formatting, incorrect urls, copy-paste mistakes, e.g. `describedBy` urls that were copies form other fields and not changed to the new document they describe.
```

> - Fixing typos, formatting, incorrect urls, copy-paste mistakes, e.g. copy-pasted `describedBy` RegEx patterns that do not point to the proper schema.

Reviewer: I understand the motivation but this is too broad and ambiguous. It could easily be misused to introduce pretty significant semantic changes. How about

```suggestion
- Cosmetic changes that don't affect which documents are considered valid against the modified schema or a schema that depends on it
```

This implies that *some* regex changes can be expedited, like changing `[abcd]` to `[a-d]` or `xx*` to `x+`, because they don't affect the set of strings that match.

And please wrap on 80 and have an empty line between list items, like in the rest of the document.

Author: Your suggestion @hannes still makes my
Pr that fixes the incorrect regex not pass the criteria for expedited process.

Reviewer: Yes, but this is the most I can agree to. Anything else would be difficult to specify without opening the door to abuse. And by the way, a more elegant approach in addressing this is to not let those typos (like the one your schema PR fixes) slip through in the first place. The inconsistency that your PR fixes could be prevented by enforcing a simple rule during the build process, or in a commit hook. So rather than spending time chasing bugs and adjusting the SOP to fast-track that chase, invest that time in build improvements.

Author: I agree. I will look into implementing such a hook on build time.

Reviewer: Note that as per SOP, reviewers resolve comments, not author.

Reviewer: > And please wrap on 80 and have an empty line between list items, like in the rest of the document.

This was not addressed.

expedited review:

- Additions of enum values to ``project.hca_bionetworks.hca_tissue_atlas``

> - Fixing typos, formatting, incorrect urls, copy-paste mistakes, e.g. `describedBy` urls that were copies form other fields and not changed to the new document they describe.

Reviewer 2: ```suggestion
- Fixing typos, formatting, incorrect urls, copy-paste mistakes, e.g. copy-pasted `describedBy` RegEx patterns that do not point to the proper schema.
```
